languages/Dockerfile.compile.j2

```diff
@@ -0,0 +1,12 @@
+FROM {{builder_image}} as builder
+USER algo
+COPY --chown=algo algosource /opt/algorithm
+RUN /opt/algorithm/bin/build
+
+FROM {{runner_image}}
+{% for artifact in config.build.artifacts %}
+COPY --from=builder --chown=algo {{artifact.source}} {{artifact.destination}}
+{% endfor %}
+USER algo
+WORKDIR /opt/algorithm
+ENTRYPOINT /bin/init-langserver
```

Comment:
Shouldn't this entrypoint be the build script? (the same is true in the other compile file)